lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -4451,6 +4452,14 @@ LogicalResult StmtEmitter::visitSV(ReturnOp op) {
   return emitOutputLikeOp(op, ports);
 }
 
+LogicalResult StmtEmitter::visitSV(IncludeOp op) {
+  startStatement();
+  ps << "`include" << PP::nbsp << "\"" << op.getTarget() << "\"";
+  emitLocationInfo(op.getLoc());
+  setPendingNewline();
+  return success();
+}
```

Comment:
There are two types of emission here. Quotes and angle brackets. They have different behavior. For completeness, can the PR add support for this in the ODS description of the op to indicate if it should be "double quote" or "angle brackets". See Section 22.4 of 1800-2023 for more information.

Reply:
Updated to include both types. I added an enum, and named the options "IncludeStyle::Local" and "IncludeStyle::System". I don't really want to call them "double-quoted" and "angle-bracketed", but I'm worried local/system are misnomers.

Reply:
I was hoping that the spec provided guidance on this, but it's not clear and seems to actually refer to them as "double quote" and "angle brackets". So, the original suggestion was regrettably intentional. 🥲 The spec does talk about the double quotes being "relative to the compiler's current working directory" and the angle brackets being "implementation dependent". You could use "standard"/"normal" and "implementation dependent".

I won't hold this up over that, I just like to have something that points directly back to what it is in the spec. And, again regrettably, the spec hasn't really named these.

Reply:
I'm just going to stick with with local/system 🤷